assets/prompt.structure.json

```diff
@@ -0,0 +1,47 @@
+{
+    "openai": {
+        "systemMessage": "You are a coding assistant API specialized in generating accurate and efficient code completions and responding in valid JSON.",
+        "complete": {
+            "format": "```complete code here```",
+            "instructions": [
+                "Do not include comments explaining the changes.",
+                "Please generate the complete code. Ensure the functionality is correct, efficient, and follows best practices. Do not provide any explanations.",
+                "Include proper indentation in code.",
+                "Use the following format to respond:"
```

Comment:
Will there be a format included? I see there's this instruction on the subsequent ones too.

Reply:
Since we'll be adding model support the instructions might be different for different models.. In future we can provide a interface to the user for adding custom instructions as well. So this structure would help.

Reply:
Thank you @AbhijeetDash!